lib/util/query_params.dart

```diff
@@ -1,20 +1,17 @@
 import 'dart:html';
 
-class QueryParams {
+/// A utility for accessing and setting query parameters.
+abstract class QueryParams {
```

Comment:
I made this `abstract` so IDEs don't think I'm constantly trying to instantiate it instead of refer to its static members.